versioned_docs/version-v2.1.5/node/custom-rpc-provider.md

```
@@ -24,19 +27,57 @@ To set up a Gnosis chain node, you will need to run both the Execution Layer (EL
+<TabItem value="RPC_with_validator" label="With validator">
 
-(**3**) Configure log size limitations (**Nethermind client v1.27.x**):
+```md
+./sedge generate full-node --execution nethermind --consensus nimbus --network=gnosis --no-mev-boost=true
+```
+</TabItem>
+</Tabs>
+
+(**4**) Configure log size limitations (**Nethermind client > v1.27.x**):
+   
    - Navigate to the "**sedge-data**" folder and open the "**.env**" file.
```

Markdownlint (0.37.0)

Expected: 0; Actual: 3
Unordered list indentation

(MD007, ul-indent)